libs/utils/utils.js

```diff
@@ -109,6 +109,8 @@ export const [setConfig, getConfig] = (() => {
       config.codeRoot = conf.codeRoot ? `${origin}${conf.codeRoot}` : origin;
       config.locale = getLocale(conf.locales);
       document.documentElement.setAttribute('lang', config.locale.ietf);
+      config.direction = (new Intl.Locale(config.locale.ietf)).textInfo.direction;
```

Comment:
I'd define the direction as a property of the `locale` object: `config.locale.direction`.

Reply:
Maybe have both?  `config.locale.direction` would take precedence but otherwise rely on `Intl.Locale` to figure out the direction.

Reply:
@jckautzmann thx for checking. makes sense. moved it.
@chrischrischris I think there was a missunderstanding. JC asked toinstead of saving the value in `config.location` to save it `config.locale.location`.

Reply:
Does anyone / anything use this config (direction / locale.direction)? We set it in the html attribute, but I don't think we should add things into the config object unless there's a service that actually needs it.

Reply:
@auniverseaway ok , removed storing in config object.